make.js

```diff
@@ -145,8 +130,8 @@ target.gendocs = function() {
 // ex: node make.js build
 // ex: node make.js build --task ShellScript
 //
-target.build = function() {
-    target.clean();
+CLI.build = function(/** @type {{ suite: string; node: string; … }} */ argv) {
+    CLI.clean(argv);
```

Comment:
```suggestion
    CLI.clean();
```
Unused parameter (previous comments)